packer/build.go

```diff
@@ -52,6 +52,63 @@ type CoreBuild struct {
 	prepareCalled bool
 }
 
+type BuildMetadata struct {
+	PackerVersion string
+	Plugins       map[string]PluginDetails
+}
+
+func (bm *BuildMetadata) GetPackerMetadata() map[string]interface{} {
```

Comment:
I'm confused, this looks essentially the same as `CoreBuild.GetMetadata` doesn't it? Why do we have both implementations? Is this used somewhere in the code?

Reply:
Fixed